ci/tasks/sonobuoy_e2e.py

```diff
@@ -106,45 +91,7 @@ def _sonobuoy(self, sonobuoy_args):
         return self._run_cmd(cmd)
 
     def _start_the_tests(self, sonobuoy_args):
-        self._sonobuoy('run ' + ' '.join(sonobuoy_args))
-
-        logger.info('Waiting for the tests to start...')
-        start_time = time.time()
-        run_time = int(time.time() - start_time)
-
-        # Shouldn't take more than 5 min for the tests to start
-        while run_time < 300:
-            try:
-                if 'Sonobuoy is still running' in self._get_status():
-                    logger.info('Tests have started!')
-                    break
-            except SonobuoyE2eTestsError:
-                pass
-
-            time.sleep(self.default_sleep)
-            run_time = int(time.time() - start_time)
-
-        if run_time >= 300:
-            raise SonobuoyE2eTestsError('Timed out while waiting for the tests to start.')
-
-    def _wait_for_the_tests(self, timeout):
-        start_time = time.time()
-        run_time = int(time.time() - start_time)
-        last_check = run_time
-        timeout = timeout * 60
-
-        while run_time < timeout:
-            # Check the status every two minutes
-            if run_time - last_check >= 120:
-                last_check = run_time
-                if 'Sonobuoy is still running' not in self._get_status():
-                    break
-
-            time.sleep(self.default_sleep)
-            run_time = int(time.time() - start_time)
-
-        if 'Sonobuoy is still running' in self._get_status():
-            raise SonobuoyE2eTestsError('Sonobuoy e2e tests ran out of time')
+        self._sonobuoy('run ' + ' '.join(sonobuoy_args) + '--wait')
```

Comment:
Can you add `--mode=certified-conformance` arg? Otherwise it's running in non-disruptive mode, which I think we don't need.
https://gitlab.suse.de/mkravec/scripts/-/blob/master/tests/sonobuoy.sh#L5

Reply:
out of scope for this card to be fair. Im all for it if it brings an improvement but it should be either discussed and a card created or a bug opened.

I think this is more like a bug innit? should run in certified confirmance but its not doing so.